README.md

```diff
@@ -1,2 +1,8 @@
 # CI
+
 CI support for Domainr
+
+Used to create: `domainr/docker-go:go1.9.4-docker17.12.0-ce` or the like
```

Comment:
This name is configured in Docker hub, right? The config there will build `Dockerfile` at the root as `:latest`.

Reply:
Yes, that was "what I was doing locally" and in fact this statement is wrong.